lib/Requestable.js

```diff
@@ -250,7 +250,13 @@ class Requestable {
 
             response.data = results;
             return response;
-         }).catch(callbackErrorOrThrow(cb, path));
```

Comment:
Why are you changing this? We should instead make the `callbackErrorOrThrow` function safe to use in this case to keep our error handling the same across pages/non-paged requests.